edit_course/operations/configure.py

```diff
@@ -490,7 +493,8 @@ def get_children(config, n = 0):
         )
 
     @staticmethod
-    def diff(old: Optional["ConfigParts"], new: "ConfigParts"):
+    def diff(old: Optional["ConfigParts"], new: "ConfigParts") -> "ConfigParts":
+        """Returns a new ConfigParts objects containing the differences between the old and new configs"""
```

Comment:
Python prefers the imperative mood in docstrings. Note also the grammatical error "a new objects" and the missing period in the end.
```suggestion
        """Return a new ConfigParts object containing the differences between the old and new configs."""
```

https://peps.python.org/pep-0257/#one-line-docstrings